ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/mapper/AmountRangeMapper.java

```diff
@@ -7,14 +7,23 @@
 @Component
 public class AmountRangeMapper implements BuilderMapper<CCDCase, AmountRange, CCDCase.CCDCaseBuilder> {
 
+    private final MoneyMapper moneyMapper;
+
+    public AmountRangeMapper(MoneyMapper moneyMapper) {
```

Comment:
If the penniestopound and poundtopennies are utility method, we dont need these injections. it can well be static methods

Reply:
These are not just utility method, these are changing to String representation as per CCD docs recommendation.